### PR TITLE
Include a docker file

### PR DESCRIPTION
**Pull request by InnocentBug**
The docker file includes both a hoomd-blue and openmm installation.
With their respective plugins to interact with pysages.

It does not support GPUs yet, but maybe we can include that in a way.

### CODE REVIEW

**Comment by InnocentBug** [comment]
warning, building the container may take a while

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+ENV CC=gcc-8
+ENV CXX=g++-8
+
+RUN python -m pip install --upgrade pip
+RUN python -m pip install jaxlib
+
+
+#HOOMD-blue dependency
+RUN git clone https://github.com/glotzerlab/hoomd-blue.git && cd hoomd-blue && git checkout v2.9.7 && mkdir build && cd build && cmake -DCMAKE_INSTALL_PREFIX=/hoomd-install -DENABLE_TESTING=OFF -DENABLE_CUDA=ON -DENABLE_MPI=ON .. && make install
```
Comment (pabloferz) [inline Dockerfile:21]:
```suggestion
RUN git clone https://github.com/glotzerlab/hoomd-blue.git && cd hoomd-blue && git checkout v2.9.7 && mkdir build && cd build && cmake -DCMAKE_INSTALL_PREFIX=/hoomd-install -DENABLE_TESTING=OFF -DENABLE_CUDA=ON -DENABLE_MPI=ON .. && make -j $(nproc) install
```

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+ENV PYTHONPATH=${PYTHONPATH}:/hoomd-install
+
+#HOOMD-blue dlext plugin
+RUN git clone https://github.com/SSAGESLabs/hoomd-dlext.git && cd hoomd-dlext && mkdir build && cd build && cmake .. && make install
+
+
+RUN apt-get update && apt-get install -y doxygen swig nvidia-cuda-dev nvidia-cuda-toolkit python3-setuptools cython3
+RUN git clone https://github.com/openmm/openmm.git &&  cd openmm &&  git checkout 7.5.0 &&  mkdir build && cd build && cmake -DPYTHON_EXECUTABLE=`which python3` -DCMAKE_INSTALL_PREFIX=../install -DBUILD_TESTING=OFF .. && make -j 6 install
```
Comment (pabloferz) [inline Dockerfile:29]:
```suggestion
RUN git clone https://github.com/openmm/openmm.git &&  cd openmm &&  git checkout 7.5.0 &&  mkdir build && cd build && cmake -DPYTHON_EXECUTABLE=`which python3` -DCMAKE_INSTALL_PREFIX=../install -DBUILD_TESTING=OFF .. && make -j $(nproc) install
```

File: Dockerfile (hunk 1 above)
Comment (pabloferz) [reply Dockerfile:21]: Maybe this will reduce the building time a bit?

**Review by pabloferz — DISMISSED** [review]

**Review by pabloferz — APPROVED** [review]

**Pull request merged**